src/components/Personalization/createExecuteDecisions.js

```diff
@@ -20,23 +20,41 @@ const buildActions = decision => {
   return decision.items.map(item => assign({}, item.data, { meta }));
 };
 
+const processMetas = (collect, logger, actionResults) => {
+  const results = flatMap(actionResults, identity);
+  const finalMetas = [];
+  const set = new Set();
```

Comment:
Will this cause issues in IE11?

Reply:
nope - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set